### PR TITLE
cart

### PR DESCRIPTION
**Pull request by aniketxparihar**
Added Cart Page Please Review!

image

### CODE REVIEW

File: cart/cart.html [hunk 1]
```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title></title>
```
Comment (shraddha15-creator) [inline cart/cart.html]: Add title

**Review by shraddha15-creator — APPROVED** [review]

File: cart/cart.css [hunk 2]
```diff
@@ -0,0 +1,156 @@
+@import url("../css/font.css");
+@import url("../css/nav.css");
+@import url("../css/card.css");
+/*Elements*/
+html {
+  width: 100%;
+  font-size: 8px;
+  font-family: Philosophy, "Noto Sans";
+}
+body {
```
Comment (KaustubhM1997) [inline cart/cart.css:10]: You can put these in main.css and use them throughout

File: cart/cart.css [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+  width: 100%;
+}
+nav {
+  display: flex;
+  flex-direction: row;
+  min-height: 10vh;
+  width: 100%;
+  align-items: center;
+  backdrop-filter: blur(10px);
+  box-shadow: -0px 0px 20px 6px rgba(0, 0, 0, 0.4);
```
Comment (KaustubhM1997) [inline cart/cart.css:24]: If you're using the box-shadow throughout your app then you can use CSS variables here and put this in the main.css file

File: cart/cart.css [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+  flex-direction: column;
+  min-width: 40rem;
+  z-index: 2;
+  display: flex;
+  flex-direction: column;
+  position: sticky;
+  top: 4vh;
+  border-radius: 8px;
+  margin-top: 4rem;
+  margin-right: 4rem;
```
Comment (KaustubhM1997) [inline cart/cart.css:62]: The shorthand property for margin can be used here

File: cart/cart.css [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+  height: max-content;
+  width: fit-content;
+  display: flex;
+  flex-direction: column;
+  justify-content: start;
+  align-items: center;
+  border-radius: 25px;
+  padding: 2rem;
+  padding-left: auto;
+  padding-right: auto;
```
Comment (KaustubhM1997) [inline cart/cart.css:86]: The padding shorthand property can be used here

File: cart/cart.css [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+  max-width: 40rem;
+  z-index: 2;
+  display: flex;
+  flex-direction: column;
+  position: sticky;
+  top: 4vh;
+  border-radius: 8px;
+  margin-top: 4rem;
+  margin-bottom: 4rem;
+  margin-right: 0rem;
```
Comment (KaustubhM1997) [inline cart/cart.css:134]: Please use the shorthand property

File: cart/cart.html [hunk 7]
```diff
@@ -0,0 +1,154 @@
...
+    <link rel="stylesheet" href="https://atomui.netlify.app/css/component.css">
+    <link rel="stylesheet" href="cart.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+</head>
+
+<body class="">
+    <nav class="">
+        <a href="../home/index.html" id="logo" class="txt-gray-200">Fitkart</a>
+        <div id="search__container">
+            <input type="text" id="search__input" placeholder="  Search Here " />
```
Comment (KaustubhM1997) [inline cart/cart.html]: You can use the input type "search" here for better accessibility.

**Review by KaustubhM1997 — APPROVED** [review]

**Pull request merged**